embedded-bins/iptables/Dockerfile

```diff
@@ -1,12 +1,14 @@
 ARG BUILDIMAGE
 FROM $BUILDIMAGE AS build
 
-RUN apk add build-base git file curl \
-	linux-headers pkgconf bison flex libmnl-dev libmnl-static libnftnl-dev
+RUN apk add build-base curl pkgconf \
+	linux-headers \
+	libmnl-dev libmnl-static \
+	libnftnl-dev
```

Comment:
I noticed this configure output

```text
checking for libnfnetlink >= 1.0... no
checking for libmnl >= 1.0... yes
checking for libnftnl >= 1.1.6... yes
checking for libnetfilter_conntrack >= 1.0.6... no
WARNING: libnetfilter_conntrack not found, connlabel match will not be built
```

```text
Iptables Configuration:
  IPv4 support:                         yes
  IPv6 support:                         yes
  Devel support:                        no
  IPQ support:                          no
  Large file support:                   yes
  BPF utils support:                    no
  nfsynproxy util support:              no
  nftables support:                     yes
  connlabel support:                    no
  profiling support:                    no
```

Are we okay with the missing connlabel support? The static libs for the needed dependencies are not in the Alpine 3.17 repos, but in edge.

Reply:
I think we want the libnetfilter_conntrack support. I believe I have seen `--label` in some iptables rule, but I don't remember where (and I cannot find it).

IPQ seems to be user space filter/mangling of packets. I don't think we need that.